UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs

```diff
@@ -179,7 +179,7 @@ private int LinkSlaves(bool relinkConnected)
 				}
 
 
-				if (slave.Master != null && relinkConnected == false) return false;
+				//if (slave.Master != null && relinkConnected == false) return false;
```

Comment:
why is this check not needed anymore?

Reply:
it seems to cause issues if stuff is partially linked, so might as well do re-complete link every time

Reply:
I see, can you remove it? Don't comment stuff out, someone in the future might be wondering why is it commented out and think uncommenting might solve something

Reply:
ok